Modules/FindCUDA/compute_arch.cmake

```diff
@@ -0,0 +1,152 @@
+# Borrowed from caffe
+# https://github.com/BVLC/caffe/blob/master/cmake/Cuda.cmake
+
+# Known NVIDIA GPU achitectures
+# This list will be used for CUDA_ARCH_NAME = All option
+SET(KNOWN_GPU_ARCHITECTURES "2.0 2.1(2.0) 3.0 3.2 3.5 5.0 5.2 5.3")
+
+################################################################################################
+# Removes duplicates from LIST(s)
+# Usage:
+#   LIST_UNIQUE(<list_variable> [<list_variable>] [...])
+MACRO(LIST_UNIQUE)
```

Comment:
I already commented on my feelings about removing this macro in the email I sent.  I think the code is clearer without it.